sts-mock/README.md

```diff
@@ -45,7 +45,7 @@ npm run lint
 To manually deploy changes made to the stack (i.e. resources or source code) to the `dev` AWS account, run the following command after assuming your credentials:
 ```shell
 sam build --parallel --cached --beta-features
-sam deploy --guided
+sam deploy --guided --capabilities CAPABILITY_NAMED_IAM
```

Comment:
Are there any specific values a dev should enter when using the guided flag? If so would it be worth adding them to the `README`?

Reply:
Can do, yes - I will add those too. I suppose those values are usually more or less the same and we get used to entering them after doing it once or twice, but it's always helpful for someone doing it the first time!